packages/core/namespace.ts

```diff
@@ -1,4 +1,18 @@
 import namespace from '@rdfjs/namespace'
+import { NamedNode } from 'rdf-js'
+
+type CubeCreatorClass =
+  'CubeProject' |
+  'ProjectsCollection' |
+  'CsvMapping' |
+  'EntryPoint'
+
+type CubeCreatorProperty =
+  'projectSourceKind' |
+  'projects' |
+  'csvMapping'
+
+type CubeCreatorNamespace = Record<CubeCreatorClass | CubeCreatorProperty, NamedNode>
```

Comment:
By making the namespace builder a fixed interface, any usage of the `cc` namespace will require only the use of terms listed above to prevent accidental typos and having to look up the actual terms every time.

WiIl also make refactoring easier